### PR TITLE
Images for feed categories - new set and missing ones

### PR DESCRIPTION
**Pull request by horus68**
Images for categories missing from code · Issue #240 · seblucas/cops -
https://github.com/seblucas/cops/issues/240

New set of images created. 64*64 px
Tested on Android on light and dark backgrounds
Color used:
Color Hex #607b8b Light Sky Blue 4 -
http://www.color-hex.com/color/607b8b

Images are created from flaticon.com
You can make new ones online from this links:
http://www.flaticon.com/free-icon/user-silhouette_64096
http://www.flaticon.com/free-icon/books-piled-_14184
http://www.flaticon.com/free-icon/buildings_34271
http://www.flaticon.com/free-icon/signing-the-contract_66720
http://www.flaticon.com/free-icon/seo-tags_15564
http://www.flaticon.com/free-icon/abc-book_73353
http://www.flaticon.com/free-icon/small-wall-clock_3921

Creative Commons BY 3.0 Copyright notice for flaticon.com and
freepik.com included in Cops (required by icons tool)

Actual look on Android:
Light background / big icons (sized by the app Moon+reader)
screenshot_2016-01-13-20-01-48

Dark background / small icons (sized by the app FBReader)
screenshot_2016-01-13-14-19-26 - copia

Note: I was not able to make the rating detail page to also use the icons.
(as discussed in https://github.com/seblucas/cops/issues/240#issuecomment-171299681 )

### CODE REVIEW

File: base.php [hunk 1]
```diff
@@ -418,7 +418,8 @@ class Entry
         Book::ALL_RECENT_BOOKS_ID    => 'images/recent.png',
         Tag::ALL_TAGS_ID             => 'images/tag.png',
         Language::ALL_LANGUAGES_ID   => 'images/language.png',
-        CustomColumn::ALL_CUSTOMS_ID => 'images/tag.png',
+        CustomColumn::ALL_CUSTOMS_ID => 'images/custom.png',
+        "cops:rating$"               => 'images/rating.png',
```
Comment (seblucas) [inline base.php]: That strange I tried to replace this line by `Rating::ALL_RATING_ID => 'images/rating.png'`

File: README [hunk 2]
```diff
@@ -73,6 +73,7 @@ It's tested by me and many other users but there's still some little bugs around
  * str_format function come from http://tmont.com/blargh/2010/1/string-format-in-php
  * All icons come from Font Awesome : http://fontawesome.github.io/Font-Awesome/
  * Thanks to all testers, translators and contributors.
+ * Feed icons made by Freepik from Flaticon website licensed under Creative Commons BY 3.0 http://www.flaticon.com and http://www.freepik.com
```
Comment (seblucas) [inline README]: Thanks for thinking about updating this file ;) You're one of the first !

**Comment by seblucas** [comment]
I made some remarks, once you test the change I propose, I'll merge it. Thanks !

File: README (hunk 2 above)
Comment (horus68) [inline README]: School librarian here!! :)

**Comment by horus68** [comment]
Fixed and working: star icons now show on rating details page too.
It was a mistake from me, I was testing "ratings" and "rating" but the test with "rating" was not proper refreshed!

**Comment by seblucas** [comment]
Thanks

**Pull request merged**